subgraphs/cbridge/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "license": "MIT",
+  "scripts": {
+    "format": "npx prettier --write ."
+  },
+  "dependencies": {
+    "@graphprotocol/graph-cli": "0.37.0",
```

Comment:
use latest npm versions

Reply:
removed. I don't think we need package.json at the subgraphs/cbridge folder.